test/unit/test-debounce.js

```diff
@@ -14,24 +14,16 @@
  * limitations under the License.
  */
 
-const mocha = require('mocha');
-const sinon = require('sinon');
-const chai = require('chai');
-const it = mocha.it;
-const describe = mocha.describe;
-const expect = chai.expect;
-const afterEach = mocha.afterEach;
-const beforeEach = mocha.beforeEach;
-const debounce = require('../utils/debounce').debounce;
+const debounce = require('../../utils/debounce');
 
-describe('debounce', function() {
+describe('debounce', () => {
```

Comment:
Mocha discourages use of arrow functions to retain access to functions like `this.skip` and `this.timeout`: https://mochajs.org/#arrow-functions